indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -425,7 +427,9 @@ <h3 class="abstract-title f-self-grow">
 
             {% if abstract.state.name not in ('submitted', 'withdrawn') %}
                 <div class="i-timeline-item">
-                    <div class="i-timeline-item-label action"></div>
+                    <div class="i-timeline-item-label action user-avatar" data-username="{{ abstract.judge.name }}">
+                        {{- abstract.judge.name[0] -}}
```

Comment:
Don't we want `display_full_name` here to get the user's preferred name display style?